client/layout/guided-tours/tours/boosted-post-tour.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import { overEvery as and } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ButtonRow,
+	LinkQuit,
+	makeTour,
+	Quit,
+	Step,
+	Tour,
+} from 'layout/guided-tours/config-elements';
+import {
+	isAbTestInVariant,
+	isEnabled,
+} from 'state/ui/guided-tours/contexts';
+import { isDesktop } from 'lib/viewport';
+
+export const BoostedPostTour = makeTour(
+	<Tour
+		name="boostedPostSurvey"
+		path="/posts/"
+		version="20170127"
+		when={ and(
+			isEnabled( 'boosted-post-survey' ),
+			isDesktop,
+			isAbTestInVariant( 'boostedPostSurvey', 'enabled' )
+		) }
```

Comment:
The triggers are really important when it comes to Guided Tours, and the framework was built specifically with that in mind. I don't think that merely popping up this tour on navigation to `/posts` is adequate. I suggest reading our extensive documentation, or — since it has very similar requisites — have a look at this [ongoing PR](https://github.com/Automattic/wp-calypso/pull/10812) by @nosolosw.

In order to have the tour start once a post is published, in a nutshell, you need to pinpoint the Redux action that interests you — action type, but other potential data in the action payload. Use the Redux dev tool for this and try publishing posts in different situations. Then you'll add the relevant action type to the GT's action log (`state.ui.actionLog`) and set up the according selector in GT; again, Andrés's PR is very similar.